### DISCUSSION TITLE
error while trying to load a model

### DISCUSSION THREAD

**Discussion by xd291** [TheLastBen/fast-stable-diffusion#1207] [Q&A]
Hi, I am getting the below error while trying to load a model (the large model box ticked in as it is 7.5 GB size)

![image](https://user-images.githubusercontent.com/121695326/210112014-c3a6c802-7821-4da1-a60c-4098b99edeeb.png)

Any ideas how to fix?

**Comment by TheLastBen** [comment]
is your sd folder recent ? if not, remove/rename it and do a fresh run

**Comment by xd291** [comment]
Thanks for the suggestion, I tried fresh intall, same error

**Reply by TheLastBen** [reply]
is the model public ? if so send the link

**Reply by xd291** [reply]
This is the one: https://civitai.com/models/2661
